lib/jose-jws-sign.js

```diff
@@ -190,7 +190,7 @@ export class Signer {
     }
 
     for (var kid in that.keyPromises) {
-      if (that.keyPromises.hasOwnProperty(kid)) {
+      if (Object.prototype.hasOwnProperty.call(that.keyPromises, kid)) {
```

Comment:
Where does this change come from? Why?

Reply:
I thought I mentioned this in the commit message, but apparently I have not. New version of ESLint comes with a [no-prototype-builtins rule](https://eslint.org/docs/rules/no-prototype-builtins) enabled and after some reading I decided it makes sense to make a code change, rather than disable the rule.